azure-devops/azext_devops/dev/boards/iteration.py

```diff
@@ -0,0 +1,207 @@
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+
+from knack.util import CLIError
+from azext_devops.devops_sdk.v5_0.work_item_tracking.models import WorkItemClassificationNode
+from azext_devops.devops_sdk.v5_0.work.models import (TeamContext,
+                                                      TeamSettingsIteration)
+from azext_devops.dev.common.arguments import convert_date_string_to_iso8601
+from azext_devops.dev.common.services import (resolve_instance_and_project,
+                                              get_work_item_tracking_client,
+                                              get_work_client)
+
+
+def get_project_iterations(depth=1, path=None, organization=None, project=None, detect=None):
+    """List iterations for a project.
+    :param depth: Depth of children to fetch.
+    :type depth: int
+    """
+    organization, project = resolve_instance_and_project(detect=detect,
+                                                         organization=organization,
+                                                         project=project)
+    client = get_work_item_tracking_client(organization)
+    list_of_iterations = client.get_classification_node(project=project,
+                                                        structure_group='iterations',
+                                                        depth=depth, path=path)
+    return list_of_iterations
+
+
+def update_project_iteration(path, child_id, name=None,start_date=None, finish_date=None,  organization=None, project=None, detect=None):
```

Comment:
will be good to add example for these
I have seen tickets which indicate that people struggle with how they enter date in CLI (mainly with format)

Reply:
Done